### PR TITLE
Culture Invariant Tests for SQL

### PR DESCRIPTION
**Pull request by RubenCerna2079**
## Why make this change?

It addresses issue #2319. Helps ensure that there are SQL tests that mirror the Cosmos tests that were added before.

## What is this change?

Added test cases for SQL to make sure datatype conversion is `CultureInfo.InvariantCulture`

## How was this tested?

- [X] Integration Tests
- [ ] Unit Tests

### CODE REVIEW

File: src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs [hunk 1]
```diff
@@ -1111,6 +1113,35 @@ public virtual async Task TestParallelDeleteMutations()
...
+                    }
+                }
+            ";
+
+            JsonElement response = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            string dbResponse = await GetDatabaseResultAsync(dbQuery);
+            using JsonDocument dbResponseJson = JsonDocument.Parse(dbResponse);
+
+            // Validate results
+            Assert.AreEqual(Convert.ToDouble(dbResponseJson.RootElement.GetProperty("total").GetDouble(), CultureInfo.InvariantCulture), response.GetProperty("total").GetDouble());
```
Comment (Aniruddh25) [inline src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs:1142]: Why do you need to convert to double if you are already doing a GetDouble() operation?

File: src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs [hunk 2]
```diff
@@ -1111,6 +1113,35 @@ public virtual async Task TestParallelDeleteMutations()
...
+
+            string graphQLMutationName = "createSales";
+            string graphQLMutation = @"
+                mutation {
+                    createSales (item: { item_name: ""test_name"", subtotal: 3.14, tax: 1.15 }) {
+                        id
+                        item_name
+                        subtotal
+                        tax
+                        total
```
Comment (Aniruddh25) [inline src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs:1132]: how is `total` calculated?

**Review by Aniruddh25 — COMMENTED** [review]
nit: briefly mention what the issue is in the `Why this change?` title

File: src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs (hunk 1 above)
Comment (RubenCerna2079) [reply src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs:1142]: Convert.ToDouble() is used to change the double value based on different windows regional format settings. The GetDouble() operation is to change the JsonElement into an object that can be used by Convert.ToDouble().

File: src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs (hunk 2 above)
Comment (RubenCerna2079) [reply src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs:1132]: `total` is calculated by adding `subtotal` and `tax`.

**Review by aaronburtle — APPROVED** [review]
LGTM!

**Pull request merged**